sap/src/main/java/nl/nn/adapterframework/extensions/sap/jco3/SapListenerImpl.java

```diff
@@ -149,36 +151,62 @@ public String getPhysicalDestinationName() {
 	}
 
 	@Override
-	public RawMessageWrapper<JCoFunction> wrapRawMessage(JCoFunction jcoFunction, PipeLineSession session) throws ListenerException {
-		return new RawMessageWrapper<>(jcoFunction, getCorrelationIdFromField(jcoFunction), null);
+	public RawMessageWrapper<M> wrapRawMessage(M jcoFunction, PipeLineSession session) throws ListenerException {
```

Comment:
Als je hier nog niet weet of het een instance van JCoFunction is, dan zou ik de parameter hier nog gewoon `message` noemen en niet `jcoFunction`.